bridetalk_project/src/components/pages/MasterLayout.jsx

```diff
@@ -3,24 +3,35 @@ import { BrowserRouter, Routes, Route } from "react-router-dom";
 import { HomePage } from "./homepage/HomePage";
 import { LoginPage } from "./login/LogInPage";
 import { CreateUserPage } from "./create-user/CreateUserPage";
+import { ResetPage } from "./reset-password/ResetPage";
+import { Footer, Navbar } from "../molecule";
+
+import "./masterlayout.css"
 
 export const MasterLayout = () => {
     return (
         <>
+        
         <BrowserRouter>
+        <div className="img">
```

Comment:
Classname="img" 😵‍💫
Please a more meaningful name for this one. 
Use classnames that perfectly describe the purpose of the elements.